src/Microsoft.TemplateEngine.Cli/TableOutput/UnicodeLength.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#nullable enable
+
+namespace Microsoft.TemplateEngine.Cli.TableOutput
+{
+    internal static class UnicodeLength
+    {
+        internal static int GetUnicodeLength (this string s)
+        {
+            return s.Sum(ch => Wcwidth.UnicodeCalculator.GetWidth((int)ch));
```

Comment:
Nit: For some reason I'm not impressed with using lambda here because perf, maybe I'm wrong... We can also improve Utf16->32 story here by doing:
```
var sum = 0;
foreach (Rune rune in s.EnumerateRunes())
{
    sum += Wcwidth.UnicodeCalculator.GetWidth(rune.Value);
}
return sum;
```

Reply:
Rune's are not available in .NET Standard, unfortunately, which we target.
Replaced the `Sum` with a for loop.